pkgs/applications/audio/lsp-plugins/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, makeWrapper
 , libsndfile, jack2
 , libGLU, libGL, lv2, cairo
-, ladspaH, php }:
+, ladspaH, php74 }:
```

Comment:
Should we replace php in the inputs rather than changing it in the file?

Reply:
Sure, moved it :slightly_smiling_face: